src/actions.rs

```diff
@@ -57,77 +57,88 @@ impl<'a> Action for Step<'a> {
+                                        .iter()
+                                        .map(|issue| IssueDecorator {
+                                            title: issue.title.clone(),
+                                            number: issue.number,
+                                            html_url: issue.html_url.clone(),
+                                            repo_name: repository
+                                                .full_name
+                                                .split("/")
+                                                .last()
+                                                .expect("Failed to split repository name")
+                                                .to_string(),
+                                            labels: issue
+                                                .labels
+                                                .iter()
+                                                .map(|l| l.name.as_ref())
+                                                .collect::<Vec<_>>()
+                                                .join(", "),
+                                            assignees: issue
+                                                .assignees
+                                                .iter()
+                                                .map(|u| u.login.as_ref())
+                                                .collect::<Vec<_>>()
+                                                .join(", "),
+                                        })
+                                        .collect();
+
+                                    results
+                                        .entry(*name)
+                                        .or_insert(Vec::new())
+                                        .extend(issues_decorator);
+                                }
+                                Err(err) => {
+                                    eprintln!("ERROR: {}", err);
+                                    err.chain()
+                                        .skip(1)
+                                        .for_each(|cause| eprintln!("because: {}", cause));
+                                    std::process::exit(1);
+                                }
                             }
-                            Err(err) => {
-                                eprintln!("ERROR: {}", err);
-                                err.chain()
-                                    .skip(1)
-                                    .for_each(|cause| eprintln!("because: {}", cause));
-                                std::process::exit(1);
+                        }
+
+                        github::QueryKind::Count => {
+                            let count = repository.get_issues_count(&gh, &query).await;
+
+                            match count {
+                                Ok(count) => {
+                                    context.insert(*name, &count);
```

Comment:
```suggestion
                                    *context.entry(*name).or_insert(0) += count;
```

I think this is wrong, right?

Reply:
yep :). The name there is the name of the query you've fired, which also matches the name of the variable used on the template.
Let's use an example, let's say that you have "p_critical" as name there and get back the number of P-critical issues. When you use 2 repos, let's say "rust-lang/rust" and "rust-lang/compiler_team" if you sum those two you would end with just one variable name that you're supposed to use in 2 templates and the number you would be getting back is the sum of the P-critical issues on both repos. What you want is to contact the name of the repo and the name of the query. In this case "rust-lang/rust.p_critical" and "rust-lang/compiler_team.p_critical" and adjust the templates so you would have the number of p_critical per repo on the right templates. We may also want to shorten those names to just "rust.p_critical" and "compiler_team.critical" for convenience.

Reply:
Got it :)

Reply:
Wait. I'm not sure. I would have *expected* the sum across all repos, I think. If I didn't want that, I would use distinct queries. I certainly expect all the issues to be grouped together into one big list (I'm not sure it's useful to separate per repo).

Reply:
If that's what we want, yeah, agreed, Niko would be right here. So, if we want to aggregate things, let's just use name, do what Niko suggested with count queries and with List queries we want to concat the resulting lists.

Reply:
Great. :)

Reply:
I think this API (`.entry`) does not exist, probably a typo